go/kbfs/libkbfs/subscription_manager.go

```diff
@@ -361,6 +465,12 @@ func (sm *subscriptionManager) SubscribeNonPath(
 
 func (sm *subscriptionManager) unsubscribePathLocked(
 	ctx context.Context, subscriptionID SubscriptionID) {
+	// First check if this is a subscription we don't yet have a folderBranch
+	// for.
+	if sm.cancelAndDeleteFolderBranchPollerLocked(subscriptionID) {
```

Comment:
This path is taken during `Shutdown` right?  So we shouldn't have to worry about lingerings I think, just wanted to make sure.

Reply:
Yes this path is taken at shutdown. All subscriptions are gone at shutdown. The "lingering" thing was referring to the subscribe/unsubscribe race.